chain/chunks/src/lib.rs

```diff
@@ -492,6 +492,7 @@ impl ShardsManager {
         network_adapter: Arc<dyn PeerManagerAdapter>,
         rng_seed: RngSeed,
     ) -> Self {
+        TransactionPool::init_metrics();
```

Comment:
this doesn't match the PR description.

I'm missing the part where you don't add transactions.